frontend/apps/reader/modules/readerfont.lua

```diff
@@ -209,11 +209,16 @@ end
 
 function ReaderFont:makeDefault(face)
     if face then
-        UIManager:show(ConfirmBox:new{
-            text = T( _("Set default font to %1?"), face),
-            ok_callback = function()
+        UIManager:show(MultiConfirmBox:new{
+            text = T( _("Set default or fallback font to %1?"), face),
```

Comment:
Keeping the current construction makes it awkward, plus I don't know if fallback font is as self-explanatory as default font. I'd suggest something like:

>Would you like %1 to be the default or the fallback font? The fallback font is used instead of the active font for missing characters.